.travis.yml

```diff
@@ -1,36 +1,22 @@
 language: python
-matrix:
-    include:
-        - python: 2.7
-          env: TOX_ENV=py27-django18
-        - python: 2.7
-          env: TOX_ENV=py27-django19
-        - python: 2.7
-          env: TOX_ENV=py27-django110
-        - python: 2.7
-          env: TOX_ENV=py27-django111
-
-        - python: 3.4
-          env: TOX_ENV=py34-django18
-        - python: 3.4
-          env: TOX_ENV=py34-django19
-        - python: 3.4
-          env: TOX_ENV=py34-django110
-        - python: 3.4
-          env: TOX_ENV=py34-django111
+dist: xenial
+sudo: true
+services:
+  - postgresql
+python:
+  - 2.7
+  - 3.5
+  - 3.6
```

Comment:
Would you consider adding support for Python 3.7 and Python 3.8?

Reply:
Done :)